src/parser/html.ts

```diff
@@ -157,6 +157,17 @@ function parseAttributeValue(
       index,
     };
   }
+  if (valueFirstChar === quote) {
```

Comment:
Wil `id="'"`(single quote in double quotes) be a problem?

Reply:
Thank you for checking this PR!
I added that test case and checked it's ok.

https://github.com/ota-meshi/svelte-eslint-parser/pull/220/files#diff-55b441b2965ba2c4cfd83a1cb1cf9ac88d9de2dd24ff63a171637ac71bbd1256R42-R47
https://github.com/ota-meshi/svelte-eslint-parser/pull/220/files#diff-4206eb687f56ab83406ab73e9e563465dd77239185a78bad5de3ca6b8b6bdc49R189-R229